docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.9'
 
 services:
   point_node:
-    image: pointnetwork/pointnetwork_node:$POINT_NODE_VERSION
+    image: pointnode-local
```

Comment:
Please avoid committing such things, otherwise it may brake the app completely for everyone.

Reply:
oops, that was occasionally, I thought I removed it